### DISCUSSION TITLE
Problem starting npx lost-pixel update

### DISCUSSION THREAD

**Discussion by HoneyQumo** [lost-pixel/lost-pixel#263] [General]
![image](https://user-images.githubusercontent.com/80570477/235598935-e6bc5c5c-190f-42ba-9509-2bac1989a6e9.png)
But after `npx playwright install` an error appears

![image](https://user-images.githubusercontent.com/80570477/235600106-c597117d-2e16-46e3-8edb-9ddc509ec313.png)
But after the recommended command `npx playwright install` the error does not disappear because `npx playwright install` installs chrome version 1060, and 1055 is required to run. How to change the launch version or change the installation version of chrome for npx playwright install?

**Comment by d-ivashchuk** [comment]
If you need to install a specific browser version, you can use Playwright's playwright-cli tool to do so. First, you'll need to install playwright-cli globally:


`npm install -g playwright-cli`

Now that you have the CLI installed, you can use the install command with the --with flag to specify the browser version you need. For example, to install Chrome 1055, you would run:


`playwright install --with=chromium:1055`

**Reply by HoneyQumo** [reply]
@d-ivashchuk, it's not working.
i did `npm install -g playwright-cli`. After I ran the command `playwright install --with=chromium:1055`, but I got an error: 
![image](https://user-images.githubusercontent.com/80570477/236122402-911d0ddc-299f-4536-9aa5-5682f0990ce1.png)

**Comment by HoneyQumo** [comment]
In general, the problem is different. If lost-pixel is based on playwright, why doesn't lost-pixel install the dependencies it needs to work?

lost-pixel requires playwright chrome 1055 version

**Comment by HoneyQumo** [comment]
@d-ivashchuk the solution you suggested doesn't work

**Comment by d-ivashchuk** [comment]
I would advice looking up how to install the correct version of playwright locally , then everything should work.
